src/bloqade/emulate/codegen/hamiltonian.py

```diff
@@ -169,10 +172,14 @@ def matrix_ele(atom_index):
             indptr[1:] = indptr[:-1]
             indptr[0] = 0
 
-            operator = csr_matrix(
+            operator = SparseMatrixCSR.create(
```

Comment:
FYI. I think most people uses `new` for constructor (so it follows the same convention in C++, rust etc.), but prob not a big deal.

Reply:
yeah this is an internal API so I don't think it matters so much.